docs/support.md

```diff
@@ -0,0 +1,33 @@
+# Customer Support
+
+In context of CDN360 we consider 3 types of customers.
+
+## Customer Types
+
+1. **Trial Customer** - Customers who onboarded through the Web UI, but have not upgraded to paid customers. These customers have a Trial credit of $50. When they reach $50 consumption, their properties may be undeployed from production automatically. To continue working with the services, an upgrade to Paid Customer is required. No credit card is required to sign-up and start using the Trial account. 
+
+    One important thing to add is that *Trial Accounts* have limitations. By default it is impossible to deploy to a production environment. We suggest setting up a property, deploying it to the staging environment and testing if it behaves properly. After that it is necessary to contact customer support to enable deployment to production.
+
+
+2. **Paid Customer** - Customers who upgraded through the Web UI using their credit card. These customers are also referred to as standard or month-to-month paid customers. They can stop using service at any time. Paid customers are obliged to pay a minimum amount of $50 per month even if the service consumption is lower than that.
+
+
+3. **Premium Customer** - Paid customers who has a particular monthly commitment and have a long-term contract with us (1Y or longer).
+
+
+## Levels of Support
+
+CDN360 offers **Standard** and **Premium** support plans. The Standard plan is offered to Trial Customers and Paid customers.  The Premium plan is offered to Premium Customers only.
+
+|item|Standard Support|Premium Support|
```

Comment:
Capitalize the column heading "item"

Reply:
For consistency, please change "and Paid customers" to "and Paid Customers."